Assets/Scripts/Bullet.cs

```diff
@@ -17,17 +23,35 @@ void Start()
     // Update is called once per frame
     void Update()
     {
-        
+        timeToDespawn -= 1;
+        if(timeToDespawn <= 0)
+        {
+            Destroy(gameObject);
+        }
     }
 
     public void FixedUpdate()
     {
-        rbody.MovePosition(rbody.position + movementVector * Time.deltaTime);
+        rbody.MovePosition(rbody.position + movementVector * Time.deltaTime * speed * shooter.shootSpeedMultiplier);
     }
 
-    public void Shoot(Vector2 movementVector)
+    public void Shoot(Vector2 movementVector, Pack shooter)
     {
         this.movementVector = movementVector;
+        this.shooter = shooter;
+        Physics2D.IgnoreCollision(GetComponent<Collider2D>(), shooter.GetComponent<Collider2D>());
```

Comment:
Seems like line 41 is unnecessary
`Physics2D.IgnoreCollision(GetComponent<Collider2D>(), this.shooter.GetComponent<Collider2D>());`